components/feature/media/src/main/java/mozilla/components/feature/media/notification/MediaNotification.kt

```diff
@@ -21,6 +22,7 @@ import mozilla.components.feature.media.ext.nonPrivateIcon
 import mozilla.components.feature.media.ext.nonPrivateUrl
 import mozilla.components.feature.media.service.AbstractMediaService
 import mozilla.components.support.base.ids.SharedIdsHelper
+import java.util.*
```

Comment:
Looks like this got included by accident

Reply:
the star import is an accident but it needs the locale package 😢